syntaxes/hubl.json

```diff
@@ -106,6 +106,10 @@
           },
           "match": "\\s*\\b(filter)\\s+([a-zA-Z_][a-zA-Z0-9_]*)\\b"
         },
+        {
+          "match": "\\b(and|else|if|in|import|(is )?not|or|recursive|with(out)?\\s+context)\\b",
+          "name": "keyword.control.hubl"
+        },
```

Comment:
I moved this up in the file so that it takes precedence over the `x is y` syntax highlighting, which was breaking the `x is not y` syntax highlight, since it was being captured as `x (is) not y` instead of `x (is not) y`. I've also added the optional group `(is )?` so it captures `is not` here.